etcdserver/api/snap/snapshotter.go

```diff
@@ -255,16 +256,19 @@ func checkSuffix(lg *zap.Logger, names []string) []string {
 
 // cleanupSnapdir removes any files that should not be in the snapshot directory:
 // - db.tmp prefixed files that can be orphaned by defragmentation
-func (s *Snapshotter) cleanupSnapdir(filenames []string) error {
+func (s *Snapshotter) cleanupSnapdir(filenames []string) (names []string, err error) {
+	names = make([]string, len(filenames))
```

Comment:
This should be `names = make([]string, 0, len(filenames))` or `names = make([]string, 0)`?

Reply:
@gyuho 
Your comment has been addressed.